e2e-tests/cypress/integration/handleSignals/samenhangMeldingen.spec.ts

```diff
@@ -0,0 +1,162 @@
+import signal_5964 from '../../fixtures/samenhang/5964.json';
+import history_7979 from '../../fixtures/samenhang/history_7979.json';
+import { SAMENHANG_TEXT } from '../../support/texts';
+import { SIGNAL_DETAILS } from '../../support/selectorsSignalDetails';
+import { SAMENHANG } from '../../support/selectorsSamenhang';
+import { MANAGE_SIGNALS } from '../../support/selectorsManageIncidents';
+
+describe('Testdata', () => {
+  it('Should setup the testdata', () => {
+    requests.createSignalSamenhang();
+    requests.createSignalSamenhang();
+    requests.createSignalSamenhang();
+  });
+});
+describe('Samenhang meldingen', () => {
+  beforeEach(() => {
+    localStorage.setItem('accessToken', generateToken('Admin', 'signals.admin@example.com'));
+  });
+  it('Should show the amount of reported signals in details of a signal', () => {
+    routes.getManageSignalsRoutes();
+    cy.visit('/manage/incidents/');
+    routes.waitForManageSignalsRoutes();
+    createSignal.openCreatedSignal();
+
+    cy.get(SIGNAL_DETAILS.labelMeldigenMelder).should('have.text', 'Meldingen van deze melder').and('be.visible');
+    cy.get(SIGNAL_DETAILS.linkMeldingenMelder).should('contain', '3 meldingen').and('be.visible');
+    cy.get(SIGNAL_DETAILS.ktoAmounts).should('contain', '0x niet tevreden').and('contain', '3x openstaand').and('be.visible');
+  });
+  it('Should open the list of signals of a reporter and go back to signal', () => {
+    cy.readFile('./cypress/fixtures/tempSignalId.json').then(json => {
+      cy.visit(`manage/incident/${json.signalId}`);
+    });
+    cy.intercept('signals/v1/private/signals/*/context/reporter?page=1**', { fixture: 'samenhang/getReporter01.json' }).as('getReporterPage1');
+    cy.intercept('signals/v1/private/signals/7979', { fixture: 'samenhang/7979.json' }).as('getSignal7979');
+    cy.intercept('signals/v1/private/signals/7979/history', { fixture: 'samenhang/history_7979.json' }).as('getHistory7979');
+    
+    cy.get(SIGNAL_DETAILS.linkMeldingenMelder).click();
+
+    cy.get(MANAGE_SIGNALS.spinner).should('not.exist');
+    cy.url().should('include', 'melder');
+    cy.contains('Meldingen van siahangsamen@sia.nl (11)').should('be.visible');
+
+    // Check signal info
+    cy.get(SAMENHANG.incidentInfo).eq(0).should('contain', signal_7979.category.sub).and('be.visible');
+    cy.get(SAMENHANG.incidentStatus).eq(0).should('contain', signal_7979.status.state_display).and('be.visible');
+    cy.get(SAMENHANG.dateTime).eq(0).should('contain', '05-05-2021 10:30').and('be.visible');
+
+    // Check KTO statuses
+    cy.get(SAMENHANG.feedbackStatus).eq(0).should('contain', 'Tevreden').and('be.visible').and('have.css', 'color', 'rgb(0, 160, 60)');
+    cy.get(SAMENHANG.feedbackStatus).eq(2).should('contain', '-').and('be.visible');
+    cy.get(SAMENHANG.feedbackStatus).eq(3).should('contain', 'Niet tevreden').and('be.visible').and('have.css', 'color', 'rgb(236, 0, 0)');
```

Comment:
There's also the possibility of _Niet ontvangen_ (see `/manage/incident/8078/melder`)